sys/cbor/cbor.c

```diff
@@ -92,6 +96,18 @@
 #endif
 
 #ifndef CBOR_NO_FLOAT
+
+/* pack to force aligned access on ARMv7 (buggy GCC) */
+#pragma GCC diagnostic error "-Wcast-align"
```

Comment:
Ahh, I like that. :+1: How much would break if we'd enable `-Wcast-align` globally?